test/parallel/xyk-pallet.API.payment-info.test.ts

```diff
@@ -131,8 +128,8 @@ test("GIVEN a paymentInfo request, WHEN extrinsic is provideLiquidityWithId THEN
 
 test("GIVEN a paymentInfo request, WHEN extrinsic is a batch with a sell/buy operation THEN non-zero is returned AND the extrinsic will fail because sell/buy are forbidden in batches tx", async () => {
   const batchAllEvent = api.tx.utility.batchAll([
-    Xyk.buyAsset(GASP_ASSET_ID, ETH_ASSET_ID, BN_HUNDRED),
-    Xyk.buyAsset(GASP_ASSET_ID, TUR_ASSET_ID, BN_HUNDRED),
+    Market.buyAsset(liqId, ETH_ASSET_ID, GASP_ASSET_ID, BN_HUNDRED),
+    Market.buyAsset(liqId, ETH_ASSET_ID, GASP_ASSET_ID, BN_HUNDRED),
```

Comment:
why are assets reversed?  before we were buying GASP and selling ETH or TUR, but now we are doing it the other way around, am I right?

Reply:
You're right, don't remember shy I did it. Fixed